### PR TITLE
support Hazelcast Management Center

### PR DESCRIPTION
**Pull request by anjia0532**
- Please make sure the below checklist is followed for Pull Requests.

require https://github.com/jhipster/jhipster-dependencies/pull/16
require https://github.com/jhipster/jhipster/pull/35

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

snipaste20171126_081022
snipaste20171126_084547
snipaste20171126_084712

snipaste20171126_090705
snipaste20171126_091804

### CODE REVIEW

File: generators/server/templates/src/main/java/package/config/_CacheConfiguration.java [hunk 1]
```diff
@@ -257,6 +257,9 @@ public HazelcastInstance hazelcastInstance(JHipsterProperties jHipsterProperties
         }
         <%_ } _%>
         config.getMapConfigs().put("default", initializeDefaultMapConfig());
+        
+        // Full reference is available at: http://docs.hazelcast.org/docs/management-center/3.9/manual/html/Deploying_and_Starting.html
+        config.setManagementCenterConfig(initializeDefaultManagementCenterConfig(jHipsterProperties))
```
Comment (PierreBesson) [inline generators/server/templates/src/main/java/package/config/_CacheConfiguration.java]: Missing a `;` here 😉

Comment (anjia0532) [reply generators/server/templates/src/main/java/package/config/_CacheConfiguration.java]: sorry,it was careless, i edit it by notepad++, can you tell me what ide do you use for `generator-jhipster` project?

**Comment by PierreBesson** [comment]
So tests will fail for now as it needs a release of jhipster-dependencies.
Regarding your comment on choice of editors, have a look at our contributing.md file.

**Comment by anjia0532** [comment]
Not only jhipster/jhipster-dependencies#16
jhipster/jhipster#35 was require too

**Comment by jdubois** [comment]
I got this working and this is really neat!!!! Thanks a lot @anjia0532 , I definitely needed this!

**Comment by anjia0532** [comment]
NOTE: 

`"Select Cluster to Connect" dialog box lists the clusters that send statistics to Management Center. You can either select a cluster to connect using the Connect button or enter your Management Center license key using the Enter License button. Management Center can be used without a license if the cluster that you want to monitor has at most 2 members.` 

by http://docs.hazelcast.org/docs/management-center/3.9/manual/html/Getting_Started.html

**Pull request merged**